Explorer/Assets/DCL/Multiplayer/SDK/Components/PlayerSDKDataComponent.cs

```diff
@@ -0,0 +1,31 @@
+using Arch.Core;
+using CommunicationData.URLHelpers;
+using CRDT;
+using DCL.ECSComponents;
+using SceneRunner.Scene;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.Multiplayer.SDK.Components
+{
+    public struct PlayerSDKDataComponent : IDirtyMarker
```

Comment:
I think in general it's too much for a single component: moreover you handle different parts of it in  different systems: it's a signal for me it should be broken down.

Reply:
Totally, the `EmoteCommand` feature made me realize that at least that one should run with a different set of systems.

I ended up with this evolved architecture that I think it's way better than before:

* Global World Systems
    * `PlayerCRDTEntitiesHandlerSystem` (propagates the new `PlayerCRDTEntity` component into the scene reserved entities)
    * `PlayerProfileDataPropagationSystem` (propagates the existent `Profile` component into the scene reserved entities)
    * `AvatarEmoteCommandPropagationSystem` (propagates the new `AvatarEmoteCommandComponent` component into the scene reserved entities)

* Scene World Systems
    * `WritePlayerIdentityDataSystem`
    * `WriteSDKAvatarBaseSystem`
    * `WriteAvatarEquippedDataSystem`
    * `WriteAvatarEmoteCommandSystem`

Let me know what you think